README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
-    <img width="200" height="200" src="https://github.com/UnchartedBull/OctoDash/raw/master/build/icon.png">
-    <br>
+    <img width="300" height="300" src="./src/assets/icon/icon-main-dark-title.svg">
```

Comment:
```suggestion
    <img width="300" height="300" src="https://github.com/UnchartedBull/OctoDash/raw/master/src/assets/icon/icon-main-dark-title.svg">
```

GitHub doesn't resolve paths AFAIK, therefore you need to include the whole URL.

Reply:
@UnchartedBull Have you tried this? When I tried with an absolute path it didn't resolve when using an svg-file even when specifying the raw output.
But if it works for you I'll of course resolve it!

Reply:
It does however resolve the relative path, you can see it in action here:

https://github.com/hessius/OctoDash

Reply:
That's funny. Maybe it does work because it is HTML and not Markdown. Pretty sure this won't work with Markdown. Then this is obsolete of course.